src/views/app/app.tsx

```diff
@@ -21,29 +23,41 @@ export function App() {
     let id = setTimeout(() => setShowSplash(false), ONE_SECOND)
     return () => clearTimeout(id)
   }, [])
+  const modalDisclosure = useDisclosure()
+  const {
+    isOpen: isAddAccountOpen,
+    onClose: onCloseAddAccount,
+    onOpen: onOpenAddAccount,
+  } = modalDisclosure
 
   if (showSplash) {
     return <Splash />
   }
   return (
-    <Routes>
-      <Route
-        path="/"
-        element={
-          <Layout>
-            <Outlet />
-          </Layout>
-        }
-      >
-        <Route index element={<Home />} />
-        <Route path="accounts" element={<Accounts />}>
-          <Route index element={<AccountsList />} />
-          <Route path=":accountAddress" element={<AccountDetails />} />
+    <>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <Layout>
+              <Outlet />
+            </Layout>
+          }
+        >
+          <Route index element={<Home modalDisclosure={modalDisclosure} />} />
+          <Route path="accounts" element={<Accounts />}>
+            <Route index element={<AccountsList />} />
+            <Route path=":accountAddress" element={<AccountDetails />} />
+          </Route>
+          <Route path="transactions/:txnId" element={<TransactionDetails />} />
+          <Route path="contacts" element={<Contacts />} />
+          <Route path="send" element={<SendAsset />} />
         </Route>
-        <Route path="transactions/:txnId" element={<TransactionDetails />} />
-        <Route path="contacts" element={<Contacts />} />
-        <Route path="send" element={<SendAsset />} />
-      </Route>
-    </Routes>
-  )
+      </Routes>
+      <AddAccountModal
+          isOpen={isAddAccountOpen}
+          onClose={onCloseAddAccount}
+      />
```

Comment:
oh jeez 😮 

I mean, I guess I asked for this when I said it was fine to do the simplest thing that could work.

Reply:
If you need details I'm free to discuss.